t04_refactored.py

```diff
@@ -135,9 +139,37 @@ def team_5_adv():
 
 
 def team_6_adv():
+    """
+
+    :return:
+    """
     pass
```

Comment:
Remove `pass`, it is a placeholder for an empty function. Now that it's not empty, remove `pass`

Reply:
I've fixed this.